app/views/finders/_spelling_suggestion.html.erb

```diff
@@ -1,11 +1,22 @@
-<% # we want an empty string if there are no suggestions on page load %> 
+<% # we want an empty string if there are no suggestions on page load %>
 <% _spelling_suggestion = '' %>
 <% if @spelling_suggestion_presenter.suggestions.any? %>
 <p class="govuk-body">Did you mean
   <% @spelling_suggestion_presenter.suggestions.each do |suggestion| %>
+    <%
+      ga4_data = {
+        module: "ga4-link-tracker",
+        ga4_link: {
+          event_name: "navigation",
+          type: "spelling suggestion",
+          section: "search",
+          text: strip_tags(suggestion[:highlighted]),
```

Comment:
Looks good - is there a reason you've used `suggestion[:highlighted]` and then stripped the tags, over just using `suggestion[:keywords]` ?

Reply:
Only because I didn't think of that. Changed 😁